src/pages/download.tsx

```diff
@@ -133,11 +133,33 @@ export default function Media() {
                   <a href="https://github.com/betaflight/blackbox-log-viewer-nightlies/releases" className="fancy-link no-underline">
                     Blackbox Viewer Nightly builds
                   </a>
-                  <div className="text-gray-400 flex flex-col mt-2">
-                    You can find release planning here:
-                    <a href="https://github.com/betaflight/betaflight/milestones" className="fancy-link no-underline">
-                      Release planning
-                    </a>
+                </div>
+              </IconElementFeature>
+
+              <IconElementFeature title="Source code" Icon={DocumentTextIcon}>
+                <div className="flex flex-col">
+                  <div className="flex flex-row space-x-1 mt-0">
+                    <span>
+                      Review and download our source code on the{' '}
+                      <a href="https://github.com/betaflight/" className="fancy-link no-underline">
+                        Betaflight GitHub Repository
+                      </a>
+                      .
+                    </span>
+                  </div>
+                </div>
+              </IconElementFeature>
+
+              <IconElementFeature title="Documentation" Icon={DocumentTextIcon}>
+                <div className="flex flex-col">
+                  <div className="flex flex-row space-x-1 mt-0">
+                    <span>
+                      Our Wiki and Development documents may be found in the{' '}
```

Comment:
```suggestion
                      The Wiki and Development documents may be found in the{' '}
```

Reply:
No need for capitalization and could be simple `Documents maybe found in the{' '}` as thinking ahead wiki and development should be consolidated into docs [documentation] over time as we considered support before.

Reply:
I'll remove 'Our' altogether.
I'm using Capitals to refer to the Proper Name of a menu item, so I use 'Wiki' when I refer to the Wiki men item, or to the betaflight Wiki as an entity, but 'wiki' when talking about wiki's generically.
How about:
`Wiki, Development and other documentation may be found in the`...